src/enigma2/data/EpgEntry.cpp

```diff
@@ -37,33 +37,35 @@ void EpgEntry::UpdateTo(EPG_TAG &left) const
   left.iFlags              = EPG_TAG_FLAG_UNDEFINED;
 }
 
-bool EpgEntry::UpdateFrom(TiXmlElement* eventNode, Channels &channels)
+bool EpgEntry::UpdateFrom(TiXmlElement* eventNode, std::map<std::string, std::shared_ptr<EpgChannel>> &m_epgChannelsMap)
```

Comment:
m_ prefix should only be used for member vars.

Reply:
Done